src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -223,93 +223,6 @@ public void SetVersion(BuildScriptGeneratorContext context, string version)
-                if (string.IsNullOrWhiteSpace(startupCommand))
-                {
-                    string mainJsFile = packageJson?.main;
-                    if (string.IsNullOrEmpty(mainJsFile))
-                    {
-                        var candidateFiles = new[]
-                        {
-                            "bin/www",
-                            "server.js",
-                            "app.js",
-                            "index.js",
-                            "hostingstart.js"
-                        };
-                        foreach (var file in candidateFiles)
-                        {
-                            if (options.SourceRepo.FileExists(file))
-                            {
-                                startupCommand = GetStartupCommandFromJsFile(options, file);
-                                _logger.LogInformation("Found startup candidate {nodeStartupFile}", file);
-                                commandSource = "CandidateFile";
-                                break;
-                            }
-                        }
-                    }
-                    else
-                    {
-                        startupCommand = GetStartupCommandFromJsFile(options, mainJsFile);
-                        commandSource = "PackageJsonMain";
-                    }
-                }
-                else
-                {
-                    if (options.SourceRepo.FileExists(NodeConstants.YarnLockFileName))
-                    {
-                        commandSource = "PackageJsonStartYarn";
-                        startupCommand = NodeConstants.YarnStartCommand;
-                        _logger.LogInformation("Found startup command in package.json, and will use Yarn");
-                    }
-                    else
-                    {
-                        commandSource = "PackageJsonStartNpm";
-                        startupCommand = NodeConstants.NpmStartCommand;
-                        _logger.LogInformation("Found startup command in package.json, and will use npm");
-                    }
-                }
-            }
-
-            if (string.IsNullOrWhiteSpace(startupCommand))
-            {
-                startupCommand = GetStartupCommandFromJsFile(options, options.DefaultAppPath);
-                commandSource = "DefaultApp";
-            }
-
-            _logger.LogInformation("Finalizing entrypoint script using {commandSource}", commandSource);
-            var templateValues = new NodeBashRunScriptProperties
-            {
-                AppDirectory = options.SourceRepo.RootPath,
-                StartupCommand = startupCommand,
-                ToolsVersions = string.IsNullOrWhiteSpace(options.PlatformVersion)
-                ? null : $"node={options.PlatformVersion}",
-                BindPort = options.BindPort
-            };
-            var script = TemplateHelpers.Render(TemplateHelpers.TemplateResource.NodeRunScript, templateValues);
```

Comment:
you can delete the resource as well (tpl file) and the entry with its name from the resource file